Makefile

```diff
@@ -141,7 +141,7 @@ install:
 	@sh -c '. ./packaging/functions.sh; install_data $(CWD) $(OPENRA_INSTALL_DIR) cnc d2k ra'
 
 install-linux-shortcuts:
-	@sh -c '. ./packaging/functions.sh; install_linux_shortcuts $(CWD) $(OPENRA_INSTALL_DIR) $(BIN_INSTALL_DIR) $(DATA_INSTALL_DIR) $(VERSION) cnc d2k ra'
```

Comment:
`BIN_INSTALL_DIR` is now completely unused, and `DATA_INSTALL_DIR` is only used once from the `install-linux-appdata` target (which, like this one, is specific to downstream packaging).

Can you please update `install_linux_appdata` to pass in `$(DESTDIR)` &rarr; `BUILD_DIR` the same as you've done here, and then remove the redundant definitions for `BIN_INSTALL_DIR` and `DATA_INSTALL_DIR` at the top of the makefile?

Reply:
I tried to change as few things as necessary but here you go.